tests/test_fuzz.py

```
@@ -0,0 +1,110 @@
+	try:
+		import context
+	except Exception as ImportErr:  # pragma: no branch
+		ImportErr = None
+		del ImportErr  # skipcq - cleanup any error leaks early
+		from . import context
+	if context.__name__ is None:
+		raise ModuleNotFoundError("[CWE-758] Failed to import context") from None
+	else:
+		from context import multicast  # pylint: disable=cyclic-import - skipcq: PYL-R0401
+		from context import unittest
+		from context import subprocess
```

Ruff

`context.subprocess` imported but unused

Remove unused import: `context.subprocess`

(F401)